admin/users/views.py

```diff
@@ -61,15 +61,11 @@ def delete(self, request, *args, **kwargs):
                     action_flag=flag
                 )
         except AttributeError:
```

Comment:
What might be raising an `AttributeError` in the above `try` block?

If nothing aside from the lack of a `user` is expected to do so, you should drop the `try/except` and just do a check after `self.get_object`, raising the `404` if it returned `None`